pcsx2-qt/Tools/InputRecording/InputRecordingViewer.cpp

```diff
@@ -57,7 +57,7 @@ QTableWidgetItem* InputRecordingViewer::createRowItem(std::tuple<bool, u8> butto
 
 void InputRecordingViewer::loadTable()
 {
-	static const auto headers = QStringList({"Left Analog", "Right Analog", "Cross", "Square", "Triangle", "Circle", "L1", "R1", "L2", "R2", "⬇️", "➡️", "⬆️", "⬅️", "L3", "R3", "Select", "Start"});
+	static const auto headers = QStringList({tr("Left Analog"), tr("Right Analog"), tr("Cross"), tr("Square"), tr("Triangle"), tr("Circle"), tr("L1"), tr("R1"), tr("L2"), tr("R2"), tr("⬇️"), tr("➡️"), tr("⬆️"), tr("⬅️"), tr("L3"), tr("R3"), tr("Select"), tr("Start")});
```

Comment:
Translating those unicode characters makes no sense to me. But I've never been a fan of using them instead of text in the first place - it assumes you have a font capable of rendering them installed.

Reply:
For this section I just went and made the whole section translatable. Though, if you'd want to change the English versions because of the Unicode, I could suggest "D-Pad Up"/"D-Pad Down"/"D-Pad Left"/"D-Pad Right".

Reply:
I'd change them. Otherwise, leave them untranslatable.

Reply:
Changed.